lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3137,7 +3137,7 @@ namespace Js
 
         } autoRestore(this->GetThreadContext());
 
-        if (!Js::Configuration::Global.EnableJitInDebugMode())
+        if (Js::Configuration::Global.EnableJitInDebugMode())
```

Comment:
I got your point. Flag vs comments did mislead me. Let me take a closer look to underlying problem.